gen/app-gen

```diff
@@ -208,9 +208,18 @@ if [ -z "${manifests_dir}" ]; then
     show_help_and_exit_error
 fi
 
-if [ ${#images[@]} -lt 1 ]; then
-    echo "No images specified. Aborting." >&2
-    show_help_and_exit_error
+if [ ${#images[@]} -lt 3 ]; then
```

Comment:
why `3`?

Reply:
I first declared `declare -a images=("")` and `declare -a images_shas=("")` which caused the number of elements to be 1 already. I changed it to be empty lists but forget to adjust the minimum number of elements here. It's an obvious error from my side :wink: